.github/workflows/scripts/dependabot-pr.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash -ex
 
-git config user.name $GITHUB_ACTOR
-git config user.email $GITHUB_ACTOR@users.noreply.github.com
+git config user.name "$GITHUB_ACTOR"
+git config user.email "$GITHUB_ACTOR"@users.noreply.github.com
```

Comment:
I think this is clearer

```suggestion
git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
```